cmd/agent/daemon/app/app.go

```diff
@@ -180,6 +180,7 @@ func (a *App) Run(ctx context.Context) error {
 		DefaultCgroupsVersion:   cgroupClient.DefaultCgroupVersion().String(),
 		ActualDestinationGetter: ct,
 		ContainerClient:         containersClient,
+		CgroupClient:            cgroupClient,
```

Comment:
Don't also need to change https://github.com/castai/kvisor/blob/main/cmd/agent/daemon/app/app.go#L192 to new signal based events?

Reply:
Good point! I've added the events directly to the map, but they should be in the system events slice.